values-changes.yaml

```diff
@@ -319,3 +319,7 @@ changes:
   - version: 32
     deletions:
       - 'teamConfig.{team}.managedMonitoring.prometheus'
+    additions:
+      - databases.gitea.imageName: ghcr.io/cloudnative-pg/postgresql:15.10
```

Comment:
The problem with adding the image specifically here is, that it will make it difficult to deliver minor updates, and also make other necessary changes to come (e.g. should the registry or repo name ever change). I would prefer only keeping the major version recorded here. We could then maintain specific images using a ClusterImageCatalog (https://cloudnative-pg.io/documentation/1.24/image_catalog/)